### ISSUE TITLE
Handling errors in Fandango's pipeline 

### ISSUE THREAD

**Issue by macagari** [fandangOrg/fandango#89]
As you know our UI is generated by sequential services, to retrieve and show information. we should decide which is the best way to handle error when one service is not working correctly. At the moment, I'm using predefined news to generate the results when that happens because I need to work on the user manual but now it is not necessary anymore.

**Comment by dmgutierrez** [comment]
We can use the status of the service to understand whether the problem is internal or whether the whole platform is down or something like that and present a simple information for the user explaining that some of the analysers or the platform itself is under maintance at this time. Also because the platform only works from 10 to 18 approximately, so any kind of process before of after that period of time will not be handle properly.

**Comment by pstalidis** [comment]
The reason that we changed the JSON that is passed from one service to the other, was to include the "status" and "message" fields.

"status" gives a code (based on http) of what happened (2xx means OK, 3xx means some kind of redirection, 4xx means this specific item failed but the platform is working, 5xx means the platform has a problem)

Also, "message" contains a textual representation of the problem that can/should be presented to the user.

If the textual messages need improvement the end-users can ask for it but for the time being, the messages that exist should be used.

**Comment by pstalidis** [comment]
This issue has been sitting here for too long and I don't think that there is any action to be done.
Can we close this?

**Comment by mmagaldi-eng** [comment]
@macagari once you've deployed the new UI with exception handling you're working on, please close this issue.

**Comment by macagari** [comment]
Yes, I had to re-define all the pipelines, I will close it when it is completely ready.
